common.sh

```diff
@@ -77,6 +57,73 @@ export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
 
 export KUBECONFIG="${SCRIPTDIR}/ocp/auth/kubeconfig"
 
+# Use a cloudy ssh that doesn't do Host Key checking
+export SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5"
+
+# Verify requisites/permissions
+# Connect to system libvirt
+export LIBVIRT_DEFAULT_URI=qemu:///system
+if [ "$USER" != "root" -a "${XDG_RUNTIME_DIR:-}" == "/run/user/0" ] ; then
+    echo "Please use a non-root user, WITH a login shell (e.g. su - USER)"
```

Comment:
Why though?

Reply:
This wasn't added in this PR tho (https://github.com/openshift-metalkube/dev-scripts/blob/2f4884a79e669a7b90655aecbaa0d60ce4b820fb/common.sh#L34-L38)